test/functional/tools/explicit_conversion.xml

```diff
@@ -0,0 +1,23 @@
+<tool id="explicit_conversion" name="explicit_conversion">
+  <command>
+    cut -f 1 '${input1_table}' > 'col_output'
```

Comment:
I don't hate it, don't say this. I just don't agree with how things went, this is a difference.
I can not judge about the technical difficulties about writing directly to the object, but the way it currently is with $output, fits to how we treat $inputs. If we want to do the shift to `from_work_dir` as recommended way, we should discuss this elsewhere and then also consider to no remove the name attribute from outputs - otherwise this is really confusing.